### PR TITLE
docs(server): doc how to run on a server as root

### LINKED ISSUE #45: Document how to run it on a server
Since this can now be launched on a server, document !

### PR DESCRIPTION
**Pull request by sbancal**
FIX #45

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -204,6 +204,45 @@ You can check it's activity by reading the `~/.enacrestic/last_backups.log` file
...
+
+```bash
+systemctl daemon-reload
+systemctl enable enacrestic.service
+systemctl start enacrestic.service
+```
+
+That will ensure enacrestic service is started when the server boot.
+
+You can know it's status with the following commands:
```
Comment (nicdub) [inline README.md:239]: You can know `its` status

**Comment by nicdub** [comment]
Hi @sbancal,
I followed the install procedure from root on Ubuntu server 20.04 and had the following error. Is there a different procedure for servers?
```
root@nicdubuntu20:~# pip3 install --user enacrestic
Collecting enacrestic
  Downloading enacrestic-0.7.0-py3-none-any.whl (9.5 MB)
     |████████████████████████████████| 9.5 MB 641 kB/s
Collecting python-pidfile
  Downloading python_pidfile-3.0.0-py3-none-any.whl (3.2 kB)
Collecting dynaconf
  Downloading dynaconf-3.1.11-py2.py3-none-any.whl (211 kB)
     |████████████████████████████████| 211 kB 51.6 MB/s
Collecting PyQt5
  Downloading PyQt5-5.15.9.tar.gz (3.2 MB)
     |████████████████████████████████| 3.2 MB 63.8 MB/s
  Installing build dependencies ... done
  Getting requirements to build wheel ... done
    Preparing wheel metadata ... error
    ERROR: Command errored out with exit status 1:
     command: /usr/bin/python3 /tmp/tmpwu1nc0vc prepare_metadata_for_build_wheel /tmp/tmpyvvjgze7
         cwd: /tmp/pip-install-me_6nhbk/PyQt5
    Complete output (31 lines):
    Traceback (most recent call last):
      File "/tmp/tmpwu1nc0vc", line 126, in prepare_metadata_for_build_wheel
        hook = backend.prepare_metadata_for_build_wheel
    AttributeError: module 'sipbuild.api' has no attribute 'prepare_metadata_for_build_wheel'

    During handling of the above exception, another exception occurred:

    Traceback (most recent call last):
      File "/tmp/tmpwu1nc0vc", line 280, in <module>
        main()
      File "/tmp/tmpwu1nc0vc", line 263, in main
        json_out['return_val'] = hook(**hook_input['kwargs'])
      File "/tmp/tmpwu1nc0vc", line 130, in prepare_metadata_for_build_wheel
        return _get_wheel_metadata_from_wheel(backend, metadata_directory,
      File "/tmp/tmpwu1nc0vc", line 159, in _get_wheel_metadata_from_wheel
        whl_basename = backend.build_wheel(metadata_directory, config_settings)
      File "/tmp/pip-build-env-r6nbcthr/overlay/lib/python3.8/site-packages/sipbuild/api.py", line 46, in build_wheel
        project = AbstractProject.bootstrap('wheel',
      File "/tmp/pip-build-env-r6nbcthr/overlay/lib/python3.8/site-packages/sipbuild/abstract_project.py", line 87, in bootstrap
        project.setup(pyproject, tool, tool_description)
      File "/tmp/pip-build-env-r6nbcthr/overlay/lib/python3.8/site-packages/sipbuild/project.py", line 585, in setup
        self.apply_user_defaults(tool)
      File "/tmp/pip-install-me_6nhbk/PyQt5/project.py", line 68, in apply_user_defaults
        super().apply_user_defaults(tool)
      File "/tmp/pip-build-env-r6nbcthr/overlay/lib/python3.8/site-packages/pyqtbuild/project.py", line 70, in apply_user_defaults
        super().apply_user_defaults(tool)
      File "/tmp/pip-build-env-r6nbcthr/overlay/lib/python3.8/site-packages/sipbuild/project.py", line 236, in apply_user_defaults
        self.builder.apply_user_defaults(tool)
      File "/tmp/pip-build-env-r6nbcthr/overlay/lib/python3.8/site-packages/pyqtbuild/builder.py", line 69, in apply_user_defaults
        raise PyProjectOptionException('qmake',
    sipbuild.pyproject.PyProjectOptionException
    ----------------------------------------
ERROR: Command errored out with exit status 1: /usr/bin/python3 /tmp/tmpwu1nc0vc prepare_metadata_for_build_wheel /tmp/tmpyvvjgze7 Check the logs for full command output.
```

**Comment by sbancal** [comment]
strange ... can you send the output of the following command: `dpkg -l | grep qt` ?

**Comment by sbancal** [comment]
On the server I used, this gives:

```
ii  libqt5core5a:amd64                    5.12.8+dfsg-0ubuntu2.1            amd64        Qt 5 core module
ii  libqt5dbus5:amd64                     5.12.8+dfsg-0ubuntu2.1            amd64        Qt 5 D-Bus module
ii  libqt5designer5:amd64                 5.12.8-0ubuntu1                   amd64        Qt 5 designer module
ii  libqt5gui5:amd64                      5.12.8+dfsg-0ubuntu2.1            amd64        Qt 5 GUI module
ii  libqt5help5:amd64                     5.12.8-0ubuntu1                   amd64        Qt 5 help module
ii  libqt5network5:amd64                  5.12.8+dfsg-0ubuntu2.1            amd64        Qt 5 network module
ii  libqt5printsupport5:amd64             5.12.8+dfsg-0ubuntu2.1            amd64        Qt 5 print support module
ii  libqt5sql5:amd64                      5.12.8+dfsg-0ubuntu2.1            amd64        Qt 5 SQL module
ii  libqt5sql5-sqlite:amd64               5.12.8+dfsg-0ubuntu2.1            amd64        Qt 5 SQLite 3 database driver
ii  libqt5svg5:amd64                      5.12.8-0ubuntu1                   amd64        Qt 5 SVG module
ii  libqt5test5:amd64                     5.12.8+dfsg-0ubuntu2.1            amd64        Qt 5 test module
ii  libqt5widgets5:amd64                  5.12.8+dfsg-0ubuntu2.1            amd64        Qt 5 widgets module
ii  libqt5xml5:amd64                      5.12.8+dfsg-0ubuntu2.1            amd64        Qt 5 XML module
ii  python3-pyqt5                         5.14.1+dfsg-3build1               amd64        Python 3 bindings for Qt5
ii  qt5-gtk-platformtheme:amd64           5.12.8+dfsg-0ubuntu2.1            amd64        Qt 5 GTK+ 3 platform theme
ii  qttranslations5-l10n                  5.12.8-0ubuntu1                   all          translations for Qt 5
```

I'd be interested to check if one of these are missing in your case ... and we should document to install.

**Comment by nicdub** [comment]
```
root@nicdubuntu20:~# dpkg -l | grep qt
ii  libqt5core5a:amd64                    5.12.8+dfsg-0ubuntu2.1            amd64        Qt 5 core module
ii  libqt5dbus5:amd64                     5.12.8+dfsg-0ubuntu2.1            amd64        Qt 5 D-Bus module
ii  libqt5gui5:amd64                      5.12.8+dfsg-0ubuntu2.1            amd64        Qt 5 GUI module
ii  libqt5network5:amd64                  5.12.8+dfsg-0ubuntu2.1            amd64        Qt 5 network module
ii  libqt5svg5:amd64                      5.12.8-0ubuntu1                   amd64        Qt 5 SVG module
ii  libqt5widgets5:amd64                  5.12.8+dfsg-0ubuntu2.1            amd64        Qt 5 widgets module
ii  libqt5x11extras5:amd64                5.12.8-0ubuntu1                   amd64        Qt 5 X11 extras
ii  qt5-gtk-platformtheme:amd64           5.12.8+dfsg-0ubuntu2.1            amd64        Qt 5 GTK+ 3 platform theme
ii  qt5dxcb-plugin:amd64                  5.0.1-3ubuntu1                    amd64        Qt platform theme integration plugin for DDE
ii  qttranslations5-l10n                  5.12.8-0ubuntu1                   all          translations for Qt 5
```

**Comment by sbancal** [comment]
Could you try to install `python3-pyqt5`

```bash
sudo apt install python3-pyqt5
```

If that works, I'll add it to the README

**Comment by nicdub** [comment]
Now it's working perfectly well! Thanks

**Review by nicdub — APPROVED** [review]
Works fine with `sudo apt install python3-pyqt5`

**Comment by sbancal** [comment]
Thanks for your helpful review : )

**Pull request merged**